backend/src/main/java/com/votogether/domain/auth/controller/AuthControllerDocs.java

```diff
@@ -58,4 +59,8 @@ ResponseEntity<ReissuedAccessTokenResponse> reissueAccessToken(
             final HttpServletResponse httpServletResponse
     );
 
+    @Operation(summary = "회원 탈퇴 하기", description = "회원 탈퇴를 한다.")
+    @ApiResponse(responseCode = "200", description = "회원 탈퇴 성공")
```

Comment:
```suggestion
    @ApiResponse(responseCode = "204", description = "회원 탈퇴 성공")
```
실제 반환되는 상태 코드랑 다른 것 같아요 !

Reply:
확인하지 못했네요..! 역시 다즐 ^^